src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -972,3 +972,81 @@
     text: az monitor metrics list-definitions --resource /subscriptions/{subscriptionID}/resourceGroups/Space1999/providers/Microsoft.Network/networkSecurityGroups/ADDS-NSG
     crafted: true
 """
+
+helps['monitor log-analytics'] = """
+type: group
+short-summary: Manage log analytics
+"""
+
+helps['monitor log-analytics workspace'] = """
+type: group
+short-summary: Manage log analytics workspace
+"""
+
+helps['monitor log-analytics workspace create'] = """
+type: command
+short-summary: Create a workspace instance
+examples:
+  - name: Create a workspace instance
+    text: az monitor log-analytics workspace create -g MyResourceGroup -n MyWorkspace
+"""
+
+helps['monitor log-analytics workspace list'] = """
+type: command
+short-summary: Get a list of workspaces under a resource group or a subscription.
+"""
+
+helps['monitor log-analytics workspace update'] = """
+type: command
+short-summary: Update a workspace instance
+"""
+
+helps['monitor log-analytics workspace pack'] = """
+type: group
+short-summary: Manage intelligent packs for log analytics workspace.
+"""
+
+helps['monitor log-analytics workspace delete'] = """
+type: command
+short-summary: Delete a workspace instance.
+"""
+
```

Comment:
If it is convenient for you, please add long summary for your commands for better understanding, But I can sccept the current scenario.

Reply:
I will choose some commands like get-schema/list-link-workspaces to add long summary. Delete is not neccessary.

Reply:
Delete list-link-workspace command and add long summary for get-schema